src/encoded/static/components/inputs/object.js

```diff
@@ -83,7 +87,7 @@ var ObjectPicker = module.exports.ObjectPicker = React.createClass({
         var previewUrl = url;
         var searchUrl = '/search/' + this.state.search;
         var actions = [
-            <button className="btn btn-primary" onClick={this.handleSelect}>Select</button>
+            <button key={1} className="btn btn-primary" onClick={this.handleSelect}>Select</button>
```

Comment:
I kept it the way it is after some experiments. This looks weird, but you need it to avoid key warnings from React. I think we can find a more “sensical” way to do this, but `PickerActions` in search.js likely needs modification for that.